_posts/2022-03-10-workflow.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "Workflow"
+date:   2022-03-18
+author: Nandi Moksnes
+category: Open Science
+tags: workflow, dataset, data, software, teaching kit, teaching material
+---
+
+<div style="text-align: justify">
```

Comment:
```suggestion
<div style="text-align: justify">
```
Will said not to use styles (or any HTML at all) - this can be done at the blog level.

Reply:
Ok, I removed it now.